cpa/policy.go

```diff
@@ -108,6 +108,14 @@ func (policy Policy) Decide(ctx context.Context, input interface{}, opts ...Eval
 		return nil, fmt.Errorf("invalid hard_fail: %w", err)
 	}
 
+	enabledHardFailRules, err := asStringSlice(org["enable_hard"])
+	if err != nil {
+		return nil, fmt.Errorf("invalid enable_hard: %w", err)
+	}
+
+	hardFailRules = append(hardFailRules, enabledHardFailRules...)
+	enabledRules = append(enabledRules, enabledHardFailRules...)
```

Comment:
We should deduplicate the the list of enabled_rules. Now that enablement can come from two difference sources, a rule could accidentally be enabled more than once, and we do not want to evaluate it twice, or have it appear twice in the resulting output.

Reply:
done - now we check if the rule exists in `enabledRules` before appending